egs/librispeech/s5/local/chain/run_tdnn.sh

```diff
@@ -18,7 +34,6 @@ remove_egs=false
 common_egs_dir=
 xent_regularize=0.1
 self_repair_scale=0.00001
```

Comment:
BTW, sed -i is useful here.  E.g.

```
find . -name '*.sh' -exec sed -i 's/foo/bar/'
```

On Tue, Aug 23, 2016 at 4:34 PM, Vijayaditya Peddinti <
notifications@github.com> wrote:

> In egs/librispeech/s5/local/chain/run_tdnn.sh
> <https://github.com/kaldi-asr/kaldi/pull/991#discussion_r75945246>:
>
> > @@ -18,7 +34,6 @@ remove_egs=false
> >  common_egs_dir=
> >  xent_regularize=0.1
> >  self_repair_scale=0.00001
>
> Could you do me a favor and modify the self_repair_scale option name in
> the make_tdnn_configs.py to self-repair-scale-nonlinearity, to remain
> consistent with make_lstm_configs.py ? This would involve modifying all the
> tdnn scripts in all recipes, but grep and sed will simplify the process
> significantly.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/991/files/51373685879bf008bd749c8351b4220e24096085..351cc3bc08279b644fc831319da940632b1eaacf#r75945246>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADJVu-adFhUUF62lB6CqXkE8BecDPujAks5qi1lYgaJpZM4Jpirx>
> .
>